tests/integration/components/country-chooser/component-test.js

```diff
@@ -0,0 +1,18 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, setupOnerror } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | country-chooser', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it throws an error if no onCountrySelection action is passed', async function(assert) {
+    setupOnerror(() => {
+      assert.throws(function() {
+        throw new Error('[component][country-chooser] Please provide a onCo')
```

Comment:
```suggestion
        throw new Error('[component][country-chooser] Please provide a onCountrySelection action')
```
no ?

Reply:
Weird that the test pass no ? :sweat_smile:

Reply:
ahaha spotted

Reply:
thanks for that, i'll make it fail first ahah